qtodotxt/lib/task_parser.py

```diff
@@ -1,15 +1,29 @@
 import re
+import string
 from datetime import datetime, date
+from qtodotxt.lib import settings
 
 
-HIGHER_PRIORITY = 'A'
-LOWER_PRIORITY = 'Z'
+HIGHEST_PRIORITY = 'A'
+LOWEST_PRIORITY = 'Z'
```

Comment:
No. These uppercase global variables are really fixed constants and must not be user editable in any way.